config/locales/en/instance_user_role_requests.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  instance_user_role_requests:
+    new:
+      header: 'Create Role Request'
```

Comment:
'Request More Permissions'

"Role Request" wouldn't mean anything to the end users.